### ISSUE TITLE
Always use Sass

### ISSUE THREAD

**Issue by jehoshua02** [deseretdigital-ui/CSS-Guidelines#20]
Even if you are just going to write vanilla css, I'd like every project setup to use `gulp-sass`. The biggest benefit is being able to easily separate styles into multiple files, which we badly need to do, especially with our new emphasis on writing everything to be reusable.

Before Closing Issue:
- [x] Response from @jstoutenburg
- [x] Response from @janiv
- [x] Response from @kbratko
- [x] Response from @jspencerDDM
- [ ] Consensus or Decision
- [ ] Pull request or other action if necessary

**Comment by rsahlstrom** [comment]
Sass should be a requirement. Gulp-sass should not be.

**Comment by jehoshua02** [comment]
In general, I think we should be using `gulp-sass`. But there are good reasons when we wouldn't use `gulp-sass`:
- If we decide to use [webpack](http://webpack.github.io/), it compiles sass already.
- If we are using [ember-cli](http://www.ember-cli.com/), it compiles sass already using [broccoli](https://github.com/broccolijs/broccoli).
- If a project is already using some other build tool.

**Comment by hhatfield** [comment]
I agree that sass is right, but the tool-chain needs to be able to vary.

**Comment by jehoshua02** [comment]
I'm fine with allowing toolchain to vary as long as build configuration is committed to the repository and the output is essentially the same.
